src/style.css

```diff
@@ -1 +1,99 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
```

Comment:
Font face for 'Avenir' has been declared multiple times with different sources. It's better to define it once and use different weights and styles as needed.